client/my-sites/checkout/cart/cart-item.jsx

```diff
@@ -51,11 +43,7 @@ export class CartItem extends React.Component {
 			return this.getFreeTrialPrice();
 		}
 
-		if (
-			cartItems.hasDomainCredit( cart ) &&
-			( isDomainProduct( cartItem ) || isDomainTransferProduct( cartItem ) ) &&
-			cartItem.cost === 0
-		) {
+		if ( cartItem.is_bundled && cartItem.cost === 0 ) {
```

Comment:
I'd probably extract this to a helper, as we did with the `hasDomainCredit` thing. There are a few more places where we show `free with plan` that need updating.

Reply:
I don't see any reason to extract this as helper as it's only used here. I found only one place that needed fixing - the Privacy Protection checkout screen. Everything else is not working with the cart so it should be fine.

Reply:
Reason is, it uses `snake_case`, and we should have functional helpers for all of these things instead of hitting the vars directly. Or at least that's how I feel. Totally ok to go like this too. :)

Reply:
Oh okay... I get it. It's just not very consistent :)

Reply:
@aidvu We have `isBundled` helper now - ready to review!